pkg/build/controller/controller_test.go

```diff
@@ -370,7 +373,14 @@ func TestHandleBuild(t *testing.T) {
 			ctrl.ImageStreamClient = tc.imageClient
 		}
 
-		err := ctrl.HandleBuild(build)
+		copy, err := kapi.Scheme.Copy(build)
+		if err != nil {
+			t.Errorf("Failed to copy object!")
```

Comment:
Please update this to something more useful. It should at least include the test number, to help you identify what went wrong, and correctly state what happened (e.g., s/object/build).